docs/src/components/Demo/cropImage.ts

```diff
@@ -30,41 +30,43 @@ export default async function getCroppedImg(
     return null
   }
 
-  const maxSize = Math.max(image.width, image.height)
-  const safeArea = 2 * ((maxSize / 2) * Math.sqrt(2))
+  // calculate bounding box of the rotated image
+  const rotRad = getRadianAngle(rotation)
+  const bBoxWidth =
+    Math.abs(Math.cos(rotRad) * image.width) + Math.abs(Math.sin(rotRad) * image.height)
+  const bBoxHeight =
+    Math.abs(Math.sin(rotRad) * image.width) + Math.abs(Math.cos(rotRad) * image.height)
```

Comment:
I was a little hesitant in making it because there's already a function called `translateSize` that does the same thing but it's much longer than my version, I wasn't exactly sure why, I thought maybe it's more precise. But if it's not, I can replace it with this version to save some space. And maybe rename it to something like `calcBoundingBoxSize`. This way in the codesandbox demos I can just import it and use it instead of writing it manually

Reply:
Replace with my version? Do you have any idea why it was created like that?

Reply:
This was contributed by another developer. I'm missing math knowledge to judge. If you have a simpler version that works, let's use it. 🙂

Reply:
I updated the helpers to use your technique as it's much shorter.